metric_providers/powermetrics/provider.py

```diff
@@ -11,6 +11,12 @@
 
 class PowermetricsProvider(BaseMetricProvider):
     def __init__(self, resolution, skip_check=False):
+        # We get this value on init as we want to have to for check_system to work in the normal case
+        pw_processes = subprocess.check_output(['pgrep', '-ix', 'powermetrics'], text=True)
+        self._pm_process_count = len(pw_processes.strip().split('\n')) if pw_processes else 0
+        print(f"NUMBER BEFORE: {self._pm_process_count}")
```

Comment:
This is very unspecifc debug message. Suggestion: "Currently already running powermetrics on the system before starting GMT powermetrics process"

Reply:
This was actually debugging for me and I forgot it. I think we should remove it as there isn't really much value in this message for the average user.

Reply:
ok with the removal., but would also be ok with the long text statement I proposed